### DISCUSSION TITLE
section columns span

### DISCUSSION THREAD

**Discussion by Ginxo** [home-assistant/frontend#21302] [Dashboard]
HI, 

sections feature is so awesome and makes more accessible for home assistant users to tweak (almost) whatever the dashboard they are able to imagine, congrats for that. 

For now I'm using it for testing purposes, since I'm not able to get what I need since I'm not able to span a section (or to span a section column) through multiple columns (maybe because my lack of knowledge...). I would like to see that feature in order to completely move myself for sections type of dashboard.

current dashboard
![image](https://github.com/home-assistant/frontend/assets/25130444/30e719a7-34d1-49f2-a038-7fe67d559a20)

vs section kind
![image](https://github.com/home-assistant/frontend/assets/25130444/06863e6f-18cf-4086-9db1-907e563b06f0)

or two columns not able to define like width
![image](https://github.com/home-assistant/frontend/assets/25130444/88e0f31a-bf0d-4c0f-9f0f-f6e153069397)

is there currently a way to actually achieve what I need here? is this a feature your are planning to publish soon? May I contribute to it? I would appreciate a bit of guidance to start working on implementing the feature.

Many thanks,
Kike.

**Comment by atomi** [comment]
Did you try the "layout" tab when you edit the section?

**Reply by Ginxo** [reply]
I did now and I moved completely to section kind of view since HA 2024.09, thanks a lot for the support

**Comment by Ginxo** [comment]
I would say this is already cover by HA 2024.09 version, very nice feature, thanks a lot!!
